### PR TITLE
Gender normalization + geo_loc_name updates

### PR DESCRIPTION
**Pull request by cassidysymons**
This PR addresses two issues:
1) It normalizes the two gender/sex columns in 10317 for the sake of generating participant reports. Once we reach critical mass with participants who answer the new question (stored in gender_v2), we'll eventually eliminate use of the sex column.
2) It updates mappings for geo_loc_name that were used to generate the last results set (August 2023) but not updated in the GitHub repo.

### CODE REVIEW

File: scripts/01.redbiom.sh [hunk 1]
```diff
@@ -32,3 +32,7 @@ redbiom fetch sample-metadata \
 python metadata_operations.py anonymize-sample-ids \
     --input-output-md ${d}/$(tag).txt \
     --input-output-tab ${d}/$(tag).biom
+
+python metadata_operations.py normalize-gender-and-sex \
+    --input-output-md ${d}/$(tag).txt \
+    --input-output-tab ${d}/$(tag).biom
```
Comment (wasade) [inline scripts/01.redbiom.sh]: Why is the feature table modified?

File: scripts/metadata_operations.py [hunk 2]
```diff
@@ -187,6 +208,34 @@ def anonymize_sample_ids(input_output_md, input_output_tab):
...
+    copy gender_v2 responses into sex for any participants who have a valid
+    response to gender_v2.
+    """
+    md = pd.read_csv(input_output_md, sep='\t', dtype=str)
+    tab = biom.load_table(input_output_tab)
+
+    mapping = dict()
+    mapping.update(_normalize_gender_and_sex(md))
+
+    tab.update_ids(mapping, inplace=True, strict=False)
```
Comment (wasade) [inline scripts/metadata_operations.py]: Sample IDs are changing in the feature table?

File: scripts/metadata_operations.py [hunk 3]
```diff
@@ -142,6 +142,27 @@ def _categories_and_types(categories):
...
+        "Female": "female",
+        "Male": "male",
+        "Not sure": "other",
+        "not provided": "not provided"
+    }
+
+    # If the DataFrame has a gender_v2 column, map it back into sex
+    if 'gender_v2' in df:
+        df['sex'] = np.where(
+            df['gender_v2'].isin(list(gender_v2_mapping.keys())),
```
Comment (wasade) [inline scripts/metadata_operations.py]:
```suggestion
            df['gender_v2'].isin(list(gender_v2_mapping)),
```

Comment (cassidysymons) [reply scripts/metadata_operations.py]: Adjusted

**Review by wasade — APPROVED** [review]

File: scripts/metadata_operations.py (hunk 2 above)
Comment (cassidysymons) [reply scripts/metadata_operations.py]: Removed

File: scripts/01.redbiom.sh (hunk 1 above)
Comment (cassidysymons) [reply scripts/01.redbiom.sh]: Removed

**Pull request merged**